src/librustc_yk_sections/emit_tir.rs

```diff
@@ -388,13 +388,12 @@ impl RenameCx {
         for y in mir.successors(n_idx) {
             // "Suppose n is the jth predecessor of y".
             let j = mir.predecessors_for(y).iter().position(|b| b == &n_idx).unwrap();
+            // "For each Phi function in y"
             for st in &mut blks[y.as_usize()].stmts {
-                match st.phi_arg_mut(j) {
-                    Some(ref mut a) => {
-                        let i = self.stack[**a as usize].last().cloned().unwrap();
-                        **a = i;
-                    },
-                    None => (), // It wasn't a Phi. Do nothing.
+                if let Some(ref mut a) = st.phi_arg_mut(j) {
+                    // We only get here if `st` was a Phi.
+                    let i = self.stack[**a as usize].last().cloned().unwrap();
```

Comment:
`let i =` is longer than `**a =`. Can we use the latter?

Reply:
We can, but I'd rather keep it as-is for parity with the algorithm in the book:

![image](https://user-images.githubusercontent.com/604955/56669086-84cbd300-66a8-11e9-824c-d7979a1b008d.png)

Reply:
OK.